lib/android/app/build.gradle

```diff
@@ -1,13 +1,21 @@
 apply plugin: 'com.android.library'
 apply from: '../prepare-robolectric.gradle'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
```

Comment:
The ternary will fail if `rootProject` doesn't have `ext` property declared.

Reply:
[`org.gradle.api.Project`](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html) extends [`org.gradle.api.plugins.ExtensionAware`](https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/ExtensionAware.html), which says:

> All extension aware objects have a special “ext” extension of type ExtraPropertiesExtension

Reply:
gotcha 👍